### ISSUE TITLE
Check for existing rebuilds for a legislature

### ISSUE THREAD

**Issue by chrismytton** [everypolitician/rebuilder#37]
In a similar spirit to https://github.com/everypolitician/countries_json_updater/issues/5 when we receive a request to rebuild a legislature we should check if there's already a rebuild queued for that legislature. If there is then we don't need to queue the rebuild again.

**Comment by tmtmtmtm** [comment]
This seems like fairly low-hanging fruit as part of speeding the rebuilder up (by reducing some rebuilds to zero!)

Other than a little bit of refactoring we should do to remove the duplication of doing this in two places, is this largely just a matter of replacing the `perform_async` calls with something that checks `RebuilderJob.jobs` first to see if there's already something with the same arguments?

**Comment by chrismytton** [comment]
> Other than a little bit of refactoring we should do to remove the duplication of doing this in two places, is this largely just a matter of replacing the perform_async calls with something that checks RebuilderJob.jobs first to see if there's already something with the same arguments?

Yep, that's pretty much it. I'll add this to my stack, should hopefully be fairly straightforward!

**Comment by chrismytton** [comment]
Pull request for this is here https://github.com/everypolitician/rebuilder/pull/54 but need to revisit it and fix the tests so that they actually fail when expected.
